### PR TITLE
WIP: Support multicore

### PR DESCRIPTION
**Pull request by tomgross**

### CODE REVIEW

File: plone-4.3.x.cfg [hunk 1]
```diff
@@ -1,2 +1,3 @@
 [buildout]
-extends = http://dist.plone.org/release/4.3.11/versions.cfg
+extends = http://dist.plone.org/release/4.3-latest/versions.cfg
```
Comment (tisto) [inline plone-4.3.x.cfg]: @tomgross please never use latest. If we do not pin specific versions the build will fail for reasons outside of the repo/code. This makes errors hard to track and fix.

**Comment by tisto** [comment]
@tomgross could you try to give me a call again? We currently might have the same problem here...

**Comment by tisto** [comment]
@tomgross could you elaborate what you want to accomplish with this pr? I just made collective.solr (master branch) work with a multi-core setup on Solr 6.4.1 without any issues. I did not use the buildout recipe though.

**Comment by tomgross** [comment]
@tisto This fork of c.solr does the following:
* Support for querying multiple cores in Plone
* Some Encoding fixes
* Interface compatible contentlisting
* Plone 5.1 compatibility (with c.indexing removed)

I probably don't have time to clean this up soon to be mergable in master.

**Pull request closed without merging**